autotools.sh

```diff
@@ -49,6 +49,11 @@ fi
 
 # m4 -- requires: nothing special
 pushd m4*
+  # texinfo uses utf-8 by default, but doc/m4.text is still iso-8859-1.
+  # MacOS sed only understands the command with the linebreaks like this.
+  sed -i '1i\
```

Comment:
I am not sure this syntax is correct on macOS.

Reply:
```bash
$ seq 5 | sed '1i\
foo
'
```
gives the right result for me on one of the CI Macs (`foo`, then the numbers 1-5).

(The single quotes are very important -- they can't be double quotes here.)

If that doesn't work for you, what error do you get? Or what should the right syntax be?

Reply:
I am worried about `-i`.

Reply:
Oh, I see. Fixed.